frontend/src/plugin/index.ts

```diff
@@ -81,10 +81,15 @@ export async function initializePlugins() {
 
   // Initialize every plugin in the order they were loaded.
   return new Promise(resolve => {
-    for (const plugin of Object.values(window.plugins)) {
+    for (const pluginName of Object.keys(window.plugins)) {
       // @todo: what should happen if this fails?
       // @todo: The return code is not checked? What is it for?
-      plugin.initialize(new Registry());
+      const plugin = window.plugins[pluginName];
+      try {
+        plugin.initialize(new Registry());
```

Comment:
Maybe move the todo comments to the top of this line now?

Reply:
Good idea. Thanks. Done.